tests/blackbox/stratis_cli_cert.py

```diff
@@ -20,12 +20,66 @@
 import time
 import unittest
 
-from testlib.utils import exec_command, exec_test_command, process_exists
+from testlib.utils import exec_command, exec_test_command, process_exists, p_n, fs_n
 from testlib.stratis import STRATIS_CLI, StratisCli, clean_up
 
 DISKS = []
 
 
+def make_test_pool():
+    """
+    Create a test pool that will later get destroyed
+    :return: Name of the created pool
+    """
+    pool_name = p_n()
+    exec_test_command([STRATIS_CLI, "pool", "create", pool_name, DISKS[0]])
+    return pool_name
+
+
+def make_test_filesystem(pool_name):
+    """
+    Create a test filesystem that will later get destroyed
+    :param pool_name: Name of a test pool
+    :return: Name of the created filesystem
+    """
+    filesystem_name = fs_n()
+    assert (
+        exec_test_command(
+            [STRATIS_CLI, "filesystem", "create", pool_name, filesystem_name]
+        )[0]
+        == 0
+    )
+    return filesystem_name
+
+
+def test_command(  # pylint: disable=bad-continuation
```

Comment:
Rename ```test_command``` to something that doesn't start with ```test``` and move the implementation inside the ```StratisCertify``` class definition. It's still an instance method of the class, which is why you pass it ```self``` so it would typically be defined in the class. But unittest likes to interpret all methods that start with ```test``` as representing tests, if they find it in a class that extends ```TestCase```, so you'll want to name the method something else to avoid that.

Then, wherever you call the method, use the ```self.test_command(...)``` syntax instead of the syntax you're using currently, ```test_command(self, ...)```.